guides/release-process.md

```diff
@@ -157,7 +153,7 @@ In the following instructions, "X.Y.Z" is used to denote the version of Cypress
 
 11. If needed, push out any updated changes to the links manifest to [`on.cypress.io`](https://github.com/cypress-io/cypress-services/tree/develop/packages/on).
```

Comment:
Is there an link to how this process works?

Reply:
The `on` package has some docs, I'm actually not sure how it works with the new docs since `sidebar` is no longer YAML, looking into it.

Reply:
cc @amirrustam

Reply:
But for now, let's not let this block this PR